lib/ast-utils/element-finder.js

```diff
@@ -29,8 +29,57 @@ var elementFinder = function elementFinder() {
   var element = null;
   var propertyHandlers = null;
 
+  /**
+   * An ES6 class is parsed differently
+   */
+  var classIsDetected = false;
+
   var visitors = {
+    /*
+     *  Set up the element when we enter the class
+     */
+    enterClassBody: function enterClassBody(node, parent) {
+      classIsDetected = true;
```

Comment:
I'd rather see this as a property of the element instead of a closure variable. This code will break if there's a non-es6 element defined in the same script as an es6 element.